spec/controllers/games_controller_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe GamesController, type: :controller do
+
+  describe "games#index action" do
+    it "should successfully show the page" do 
+      get :index
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "games#new action" do 
+    it "should sucessfully show the new form" do
```

Comment:
Your test names are a bit confusing because the tests don't actually check whether a particular form or page is being shown. Instead, they're just looking for the server response to include a certain status. Could you rename them accordingly?